lib/common.bash

```diff
@@ -49,6 +52,27 @@ check_processes() {
 	done
 }
 
+# Checks that pods were not left
+check_pods() {
+	if [ -d ${VC_POD_DIR} ]; then
+		# Verify that pods were not left
+		pods_number=$(ls ${VC_POD_DIR} | wc -l)
+		if [ ${pods_number} -ne 0 ]; then
+			die "${pods_number} pods left and found at ${VC_POD_DIR}"
+		fi
+	else
+		echo "Not ${VC_POD_DIR} directory found"
```

Comment:
- s/Not/No/
- Shouldn't this be a `die` scenario?

Reply:
True, but we're running `check_pods()` after the tests have run so we would expect that directory to exist at that point wouldn't we?

Reply:
Today, that is probably true. OK, if the first batch of tests maybe only test container creation failure cases, or in the future we have a test that does a docker/kata full teardown and reconfig, then the premise may not be true.
We could set it to a `die` right now, and spin the CI to see how it goes. Later on it would be obvious that it needs changing if necessary, as the CIs would fail on the PR making any relevant change (one hopes ;-) ).
So, I'm fine making it a `die` right now it that passes the CIs.